docs(generate): add nojekyll option in generator (#1280)
nuxt/nuxtjs.org: content/en/_archives/api/configuration-generate.md

- **Deprecated!**
- Use [build.html.minify](/api/configuration-build#html-minify) instead

## nojekyll

- Type: `Boolean`
- Default: `true`

> By default, adding `.nojekyll` file for serving static pages on GitHub Pages. If you want to disable this feature you can set the value to `false`

Reviewer: ```suggestion
Adds a `.nojekyll` file to enable serving `/_nuxt/` when deployed on GitHub Pages. If you want to disable this feature you can set the value to `false`.
```

Author: Thank you for your suggestion. I committed it.
